backend/api_v2/exceptions.py

```
@@ -56,3 +56,36 @@ def __init__(
+    
+    def __init__(
+        self,
+        url: str = "",
+        error_message: str = "",
+        status_code: int = 400,
+        detail: str | None = None,
+        code: str | None = None,
+    ):
+        if detail is None:
+            detail = f"Failed to fetch file from URL: {url}. Error: {error_message}"
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)